pkgs/applications/window-managers/phosh/default.nix

```diff
@@ -88,7 +100,9 @@ in stdenv.mkDerivation rec {
 
   postUnpack = ''
```

Comment:
`fetchFromGitLab` now supports `fetchSubmodules`: https://github.com/NixOS/nixpkgs/pull/131047

Reply:
Cool, that makes things even simpler then :+1: